AGENT.md

```diff
@@ -0,0 +1,7 @@
+# Agent
```

Comment:
I think it should be agents.md? https://github.com/sst/opencode/issues/2034#issuecomment-3222998718